bower.json

```
@@ -1,6 +1,6 @@
 {
   "name": "adapt-navigationLogo",
-  "version": "3.3.0",
+  "version": "3.4.0",
```

```suggestion
  "version": "3.3.0",
```

Please revert this, it's done with automation.